Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
```

Comment:
What even is the purpose of this file if there is an Ubuntu 20.04 docker file in `docker/build_and_test` [here](https://github.com/helibproject/HElib/blob/master/docker/build_and_test/Dockerfile.UBUNTU)?

Reply:
there was an original dockerfile in the top directory .... I can just delete it.

Reply:
I don't think it is necessary anymore unless it is doing something that the one in `docker/build_and_test` cannot do.